contracts/Dependencies/ERC20Permit.sol

```diff
@@ -43,7 +43,7 @@ abstract contract ERC20Permit is ERC20, IERC2612Permit {
 		uint8 v,
 		bytes32 r,
 		bytes32 s
-	) public virtual override {
```

Comment:
This is public in the OpenZeppelin implementaion

Reply:
Yes, I know the implementation, but why can't it be external?
Likewise, in this case, whether it is public or external does not make a difference with respect to the gas